gym-unity/gym_unity/tests/test_gym.py

```diff
@@ -97,54 +97,49 @@ def create_mock_brainparams(
     Creates a mock BrainParameters object with parameters.
     """
     # Avoid using mutable object as default param
-    if vector_action_space_size is None:
-        vector_action_space_size = [2]
-    mock_brain = mock.Mock()
-    mock_brain.return_value.number_visual_observations = number_visual_observations
-    if number_visual_observations:
-        mock_brain.return_value.camera_resolutions = [
-            CameraResolution(width=8, height=8, num_channels=3)
-            for _ in range(number_visual_observations)
-        ]
-
-    mock_brain.return_value.vector_action_space_type = vector_action_space_type
-    mock_brain.return_value.vector_observation_space_size = (
-        vector_observation_space_size
-    )
-    mock_brain.return_value.vector_action_space_size = vector_action_space_size
-    return mock_brain()
-
-
-def create_mock_vector_braininfo(num_agents=1, number_visual_observations=0):
+    act_type = ActionType.DISCRETE
```

Comment:
nit: update "Creates a mock BrainParameters object with parameters" docstring above. 

+1 to using an actual object and not mock here.